src/AppAny.Quartz.EntityFrameworkCore.Migrations/Quartz/QuartzCalendar.cs

```diff
@@ -1,4 +1,4 @@
-namespace AppAny.Quartz.EntityFrameworkCore.Migrations.PostgreSQL
+namespace AppAny.Quartz.EntityFrameworkCore.Migrations.Quartz
```

Comment:
Could you please remove this double-quartz namespace? It is part of `AppAny.Quartz.EntityFrameworkCore.Migrations` assembly with `AppAny.Quartz.EntityFrameworkCore.Migrations` namespace, just a core model

Reply:
ok np